lisa/util/__init__.py

```diff
@@ -598,6 +601,98 @@ def is_valid_url(url: str, raise_error: bool = True) -> bool:
     return is_url
 
 
+def _raise_or_log_failure(log: Logger, raise_error: bool, failure_msg: str) -> bool:
+    if raise_error:
+        raise LisaException(failure_msg)
+    else:
+        log.info(failure_msg)
```

Comment:
It should be debug level by default, because it's not worth to raise an exception. It means it's ignorable error.